src/main/resources/component-web.conf

```diff
@@ -411,18 +411,41 @@ scripting {
             "java.lang.String.*" = true
             "java.lang.Object.*" = true
             "java.lang.Enum.*" = true
+            "java.lang.Class.getSimpleName" = true
+            "java.lang.Throwable.getLocalizedMessage" = true
+            "java.time.LocalDate.*" = true
             "java.util.List.*" = true
             "java.util.Collection.*" = true
+            "java.util.Collections.emptyList" = true
+            "java.util.Optional.*" = true
+            "java.util.Map.*" = true
             "java.util.Set.*" = true
+            "java.util.stream.Stream.*" = true
+            "sirius.kernel.commons.MultiMap.get" = true
+            "sirius.kernel.commons.MultiMap.unmodifiableKeySet" = true
+            "sirius.kernel.commons.MultiMap.values" = true
             "sirius.kernel.commons.Tuple.*" = true
+            "sirius.kernel.commons.Tuple.setFirst" = false
+            "sirius.kernel.commons.Tuple.setSecond" = false
             "sirius.kernel.commons.Value.*" = true
             "sirius.kernel.commons.Amount.*" = true
             "sirius.kernel.commons.Strings.*" = true
+            "sirius.kernel.settings.ExtendedSettings.getExtension" = true
             "sirius.kernel.settings.Settings.get" = true
+            "sirius.kernel.settings.Settings.getSettings" = true
+            "sirius.kernel.settings.Settings.getString" = true
+            "sirius.kernel.settings.Settings.getStringList" = true
+            "sirius.kernel.settings.Settings.getTranslatedString" = true
+            "sirius.kernel.Sirius.getSettings" = true
```

Comment:
I'd not reveal the complete system config to a sandboxed user - there is way too much interesting stuff in there...

Reply:
Yeah, of course. That's what reviews are for 😅